static/js/settings_invites.js

```diff
@@ -50,6 +50,7 @@ function populate_invites(invites_data) {
         name: 'admin_invites_list',
         modifier: function (item) {
             item.invited_absolute_time = timerender.absolute_time(item.invited * 1000);
+            item.is_admin = page_params.is_admin;
```

Comment:
Do we need to add this to the `item`?  Seems a little hacky.  @showell may know a better way.

Reply:
@timabbott It is indeed hacky to write `is_admin` to the `item` here, but at least `item` here is a pretty ephemeral thing that just comes from the server response (see `on_load_success`), so I don't feel bad about modifying it.  Obviously I would be more concerned if this were some kind of `people` object or something.

I think we're ok here.

But I also could see just creating a new data structure for the render piece:

~~~ js
modifier: function (server_data) {
   const item = {...server_data};
   // ....
~~~

I'm fine either way--it's probably easiest to stick with what we have now.

Reply:
sgtm.